docs(tutorial): update toh-pt5 wording

3ygun opened the pull request:
# Goal

I was reading sentence by sentence and stopped after this one
as I was getting an error ("Cannot find primary outlet") and
not a displayed visualization.
This change would have resolved the issue for me and
improves the flow into the "However,..." sentence.

# Angular Questions

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [N/A] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)


**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

- Slightly confusing toh-pt5 docs

**What is the new behavior?**

- Better confusing toh-pt5 docs

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...


**Other information**:

3ygun committed:
docs(tutorial): update toh-pt5 wording

I was reading sentence by sentence and stopped after this one
as I was getting an error ("Cannot find primary outlet") and
not a displayed visualization.
This change would have resolved the issue for me and
improves the flow into the "However,..." sentence.

gkalpak commented on aio/content/tutorial/toh-pt5.md:
```diff
@@ -352,7 +352,7 @@ performs the initial navigation based on the current browser URL.
 
 If you paste the path, `/heroes`, into the browser address bar at the end of the URL,
-the router should match it to the `heroes` route and display the `HeroesComponent`.
+the router should match it to the `heroes` route and retrieve the `HeroesComponent`.
```
FWIW, the previous wording is better imo. "retrieve" feels like a whole new concept.

3ygun replied:
My main problem with "display" is that, if you've only completed the tutorial until this point, it doesn't get displayed (Instead giving you an error). After, one adds line 357 (e.g. `<router-outlet>`) it does get displayed BUT, not until then.

Furthermore, how does this use of the word display here fit with the one on line 356? I can see the correlation in hindsight but spent ~5 minutes the first time through the tutorial. 😞 

The [RouterLink](https://github.com/angular/angular/blob/master/packages/router/src/directives/router_link.ts) source sounds like "link in" could also be used instead of "retrieve".

dherges replied:
yes, it's should not a "retrieve" but more of a "resolve the `ComponentFactory<HeroesComponent>`, create a new component instance and attach it to the DOM". right?

3ygun replied:
TBH don't know about the `ComponentFactory<HeroesComponent>` part. But overall that is the idea I was trying to get across. I think "resolve to" would be cleaner than "retrieve" now that you mention it.

Should I push the change as a new commit on this PR or make a new PR with the updated wording and link in the comments to this one?

dherges replied:
"..and render a `HeroesComponent` instance" ? wdyt?

The paragraph would be:

"If you paste the path, `/heroes`, into the browser address bar at the end of the URL, the router should match it to the `heroes` route and render a `HeroesComponent` instance [optional: to the screen]. However, you have to tell the router where to display the component. To do this, you can add a `<router-outlet>` element at the end of the template. `RouterOutlet` is one of the directives provided by the `RouterModule`. The router displays each component immediately below the `<router-outlet>` as users navigate through the app."

3ygun replied:
Made the change in the commit below. Sorry for the delay!

3ygun committed:
docs(tutorial): update toh-pt5 wording

Clarified that the render optionally displays the content on screen. Updated from Angular team comments.

3ygun commented:
Closing. Don't think it's relevant anymore.
